app/models/lookup/import/adviser.rb

```diff
@@ -0,0 +1,26 @@
+module Lookup
+  module Import
+    class Adviser < ActiveRecord::Base
+      validates_length_of   :reference_number, is: 8
+      validates_presence_of :name
+
+      def self.table_name
+        "lookup_import_#{super}"
+      end
+
+      def self.fca_import_copy_statement
+        "COPY #{table_name} (reference_number, name, created_at, updated_at) FROM stdin;"
+      end
+
+      def self.truncate_sql
+        "TRUNCATE #{table_name};"
```

Comment:
We can, however that will put the truncate in a different transaction to the copy.